README.rst

```diff
@@ -331,79 +333,79 @@ We assume you have an instance (or more) of Vault running and you have privilege
+
+    $ # It is required to enable a transit engine if not already done (It is suggested to create a transit engine specifically for SOPS, in which it is possible to have multiple keys with various permission levels)
+    $ vault secrets enable -path=sops transit
+    Success! Enabled the transit secrets engine at: sops/
+
+    $ # Then create one or more keys
+    $ vault write sops/keys/firstkey type=rsa-4096
+    Success! Data written to: sops/keys/firstkey
+
+    $ vault write sops/keys/secondkey type=rsa-2048
+    Success! Data written to: sops/keys/secondkey
 
-	$ # Substitute this with the address Vault is running on
-	$ export VAULT_ADDR=http://127.0.0.1:8200 
-
-	$ # this may not be necessary in case you previously used `vault login` for production use
-	$ export VAULT_TOKEN=toor 
-	
-	$ # to check if Vault started and is configured correctly
-	$ vault status
-	Key             Value
-	---             -----
-	Seal Type       shamir
-	Initialized     true
-	Sealed          false
-	Total Shares    1
-	Threshold       1
-	Version         1.2.0
-	Cluster Name    vault-cluster-618cc902
-	Cluster ID      e532e461-e8f0-1352-8a41-fc7c11096908
-	HA Enabled      false
-
-	$ # It is required to enable a transit engine if not already done (It is suggested to create a transit engine specifically for sops, in which it is possible to have multiple keys with various permission levels)
-	$ vault secrets enable -path=sops transit
-	Success! Enabled the transit secrets engine at: sops/
-
-	$ # Then create one or more keys
-	$ vault write sops/keys/firstkey type=rsa-4096
-	Success! Data written to: sops/keys/firstkey
-
-	$ vault write sops/keys/secondkey type=rsa-2048
-	Success! Data written to: sops/keys/secondkey
-
-	$ vault write sops/keys/thirdkey type=chacha20-poly1305
-	Success! Data written to: sops/keys/thirdkey
-
-	$ sops --encrypt --hc-vault-transit $VAULT_ADDR/v1/sops/keys/firstkey vault_example.yml
-
-	$ cat <<EOF > .sops.yaml
-	creation_rules:
-		- path_regex: \.dev\.yaml$
-		  hc_vault_transit_uri: "$VAULT_ADDR/v1/sops/keys/secondkey"
-		- path_regex: \.prod\.yaml$
-		  hc_vault_transit_uri: "$VAULT_ADDR/v1/sops/keys/thirdkey"
-	EOF
-
-	$ sops --verbose -e prod/raw.yaml > prod/encrypted.yaml
+    $ vault write sops/keys/thirdkey type=chacha20-poly1305
+    Success! Data written to: sops/keys/thirdkey
+
+    $ sops --encrypt --hc-vault-transit $VAULT_ADDR/v1/sops/keys/firstkey vault_example.yml
+
+    $ cat <<EOF > .sops.yaml
+    creation_rules:
+        - path_regex: \.dev\.yaml$
+          hc_vault_transit_uri: "$VAULT_ADDR/v1/sops/keys/secondkey"
+        - path_regex: \.prod\.yaml$
+          hc_vault_transit_uri: "$VAULT_ADDR/v1/sops/keys/thirdkey"
+    EOF
+
+    $ sops --verbose -e prod/raw.yaml > prod/encrypted.yaml
 
 Adding and removing keys
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-When creating new files, ``sops`` uses the PGP, KMS and GCP KMS defined in the
+When creating new files, SOPS uses the PGP, KMS and GCP KMS defined in the
```

Comment:
As we later make mention of flags, maintaining the command reference is better here I think.

There are a couple of other places where this applies as well.

Reply:
I hope I manged to fix the right ones you meant.